pep-0678.rst

```diff
@@ -8,7 +8,7 @@ Content-Type: text/x-rst
 Requires: 654
 Created: 20-Dec-2021
 Python-Version: 3.11
-Post-History:
+Post-History: 2022-01-27
```

Comment:
There's no formal requirement I can find in PEP-0001, but less than a week of discussion and review between announcing/opening a discussion thread for a PEP and submitting it for final consideration for the SC seems _awfully_ short, at least to me...